cfgov/jinja2/v1/_layouts/base-common.html

```diff
@@ -47,11 +46,8 @@
 
     <title>
         {%- block title -%}
-            {%- if page -%}
-                {{ page.seo_title or page.title }} | Consumer Financial Protection Bureau
-            {%- else -%}
-                Consumer Financial Protection Bureau
-            {%- endif %}
+            {%- if page -%} {{ page.seo_title or page.title }} | {%- endif %}
```

Comment:
I think you want the `endif` like this to preserve the space after the pipe when it's true, but have no space before "Consumer" when it's false: `{% endif -%}

Reply:
Good catch! Fixed in https://github.com/cfpb/cfgov-refresh/pull/3902/commits/990568f0bc51964c10315a204dc99868de7f4dcc